src/CorrugatedIron.Tests/Models/MapReduce/Inputs/MapReduceSerializationTestsBase.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2011 - OJ Reeves & Jeremiah Peschka
+// Copyright (c) 2015 - Basho Technologies, Inc.
+//
+// This file is provided to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file
+// except in compliance with the License.  You may obtain
+// a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace CorrugatedIron.Tests.Models.MapReduce.Inputs
+{
+    public abstract class MapReduceSerializationTestsBase
+    {
+        protected static string Serialize(Func<JsonWriter, JsonWriter> doWrite)
+        {
+            var sb = new StringBuilder();
+
+            using (var sw = new StringWriter(sb))
```

Comment:
On the double `using`?  Sure, but we've got a whole bunch of those to fix and finalize correctly.  I'll open an issue about those, b/c they're littered everywhere.